provisions/roles/jenkins/slave/tasks/setup_worker.yml

```diff
@@ -19,48 +19,49 @@
   file: dest=/opt/cccp-service/beanstalk_worker state=directory
 
 - name: Build container-pipeline image
-  shell: docker build -t container-pipeline -f ./Dockerfile .
-  args:
-      chdir: /opt/cccp-service
+  sudo: yes
```

Comment:
Do you mean that if a dev is allowed to login to system X as user `dev` and if the user `dev` doesn't have `sudo` privileges on that system, the `sudo: yes` directive will help bypass that and start the container?